bitcoin/src/address/mod.rs

```diff
@@ -528,8 +528,8 @@ impl Address {
     pub fn script_pubkey(&self) -> ScriptBuf {
         use AddressInner::*;
         match self.0 {
-            P2pkh { ref hash, network: _ } => ScriptBuf::new_p2pkh(hash),
-            P2sh { ref hash, network: _ } => ScriptBuf::new_p2sh(hash),
+            P2pkh { ref hash, network: _ } => ScriptBuf::new_p2pkh(*hash),
+            P2sh { ref hash, network: _ } => ScriptBuf::new_p2sh(*hash),
```

Comment:
In b810f1b92accb456699167502ba923583947822e:

Can just drop the `ref` here rather than adding a `*`.

Reply:
Done as suggested.